### PR TITLE
Add info for the PoW server discontinuation

### PR DESCRIPTION
**Pull request by thsfs**

### CODE REVIEW

**Comment by thsfs** [comment]
In network-upgrades.md file, it still requires changing the related date to the correct value once we know the correct date. Also, in the changed table there are some TBD entries that seem to be old info that was never defined. It requires a review.

image

File: docs/releases/network-upgrades.md [hunk 1]
```diff
@@ -25,13 +25,14 @@ To help ensure Quality of Service on the network by providing the ability to gen
...
+| TBD                                      | <span class="no-break">Epoch blocks start</span> | Distribution of epoch blocks to each account which upgrades them to use the new PoW algorithm.  Once an account is upgraded nodes will only validate work made using the new PoW algorithm for that account.                                                                                                                                                                                                                                                                       |
+| TBD                                      | Epoch blocks end | Distribution of epoch blocks ends after all accounts are upgraded.                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (gr0vity-dev) [inline docs/releases/network-upgrades.md:35]: Is a new PoW algorithm still planned ?
Shouldn't we rather talk about the bounded backlog and the featuresrequired to reach "commercial grade"

Comment (thsfs) [reply docs/releases/network-upgrades.md:35]: Yes, just realized that the changes proposed here are incorrect. What happened is that we only stopped bundling the PoW server in the Docker container, that was the old implementation written in C++.

**Comment by thsfs** [comment]
The changes proposed by this PR are incorrect, closing it.

**Pull request closed without merging**